components/private/Snapshot/BUILD

```diff
@@ -38,11 +38,32 @@ swift_library(
     ],
 )
 
+mdc_objc_library(
+    name = "Supplemental",
+    testonly = 1,
+    srcs = glob(["src/Supplemental/*.m"]),
+    hdrs = glob(["src/Supplemental/*.h"]),
+    visibility = ["//visibility:public"],
+)
+
+mdc_objc_library(
+    name = "Implementation",
```

Comment:
Hrm; Implementation is a bit confusing to me here because my expectation is that the "Snapshot" target is the implementation. What is the intent of this target?